<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Consultas</class>
 <widget class="QWidget" name="Consultas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QToolBox" name="toolBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>720</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
    <number>5</number>
   </property>
   <widget class="QWidget" name="page_19">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>540</height>
     </rect>
    </property>
    <attribute name="label">
     <string> Ventas por período por estación y por tipo de combustible.</string>
    </attribute>
    <widget class="QTableView" name="table_VentasPeriodo_Page1">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>20</y>
       <width>721</width>
       <height>481</height>
      </rect>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Sunken</enum>
     </property>
     <property name="textElideMode">
      <enum>Qt::TextElideMode::ElideMiddle</enum>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="dateTimeEdit_FechaInicio_Page1">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>120</y>
       <width>194</width>
       <height>25</height>
      </rect>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="dateTimeEdit_FechaFin_Page1">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>170</y>
       <width>194</width>
       <height>25</height>
      </rect>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_97">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Fecha Inicio:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_98">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Fecha Fin:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_99">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Estacion:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_Estacion_Page1">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_Combustible_Page1">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>70</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_100">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Combustible:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BotonConsultar_Page1">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>250</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Consultar</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_131">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>410</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Suma de  Montos:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_132">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>460</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Suma de Litros:</string>
     </property>
    </widget>
    <widget class="QLabel" name="SumaVentas_label">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>410</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::TextFormat::AutoText</enum>
     </property>
    </widget>
    <widget class="QLabel" name="SumaLitros_Label">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>460</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::TextFormat::AutoText</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_20">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>540</height>
     </rect>
    </property>
    <attribute name="label">
     <string>Comparativo entre estaciones</string>
    </attribute>
    <widget class="QPushButton" name="BotonConsultar_Page2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>220</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Consultar</string>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="dateTimeEdit_FechaFin_Page2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>90</y>
       <width>194</width>
       <height>25</height>
      </rect>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_101">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Fecha Fin:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_102">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Fecha Inicio:</string>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="dateTimeEdit_FechaInicio_Page2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>194</width>
       <height>25</height>
      </rect>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_103">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ranking por:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_Ranking_Page2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>150</y>
       <width>191</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableView" name="table_ComparativoEstaciones_Page2">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>20</y>
       <width>641</width>
       <height>471</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_21">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>540</height>
     </rect>
    </property>
    <attribute name="label">
     <string>Precios vigentes para una fecha dada por estación y combustible</string>
    </attribute>
    <widget class="QLabel" name="label_104">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Estacion:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_Estacion_Page3">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_105">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Combustible:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_Combustible_Page3">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>90</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_106">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Fecha:</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit_Page3">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>150</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="PrecioVigente_Label_Page3">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>80</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::TextFormat::AutoText</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="BotonConsultar_Page3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>240</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Consultar</string>
     </property>
    </widget>
    <widget class="QLabel" name="PrecioVigente_Label">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>80</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Precio Vigente:</string>
     </property>
     <property name="textFormat">
      <enum>Qt::TextFormat::AutoText</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_22">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>540</height>
     </rect>
    </property>
    <attribute name="label">
     <string> Top de combustibles por estación en un rango de fechas</string>
    </attribute>
    <widget class="QComboBox" name="comboBox_Estacion_Page4">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_107">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Fecha Inicio:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_108">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Fecha Fin:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_109">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Estacion:</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit_Page4">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>70</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit2_Page4">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>120</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="BotonConsultar_Page4">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>240</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Consultar</string>
     </property>
    </widget>
    <widget class="QTableView" name="table_ComparativoEstaciones_Page4">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>40</y>
       <width>701</width>
       <height>441</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_69">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ranking por:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_Ranking_Page4">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>180</y>
       <width>191</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_23">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>540</height>
     </rect>
    </property>
    <attribute name="label">
     <string>Margen sencillo por combustible</string>
    </attribute>
    <widget class="QComboBox" name="comboBox_Combustible_Page5">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>110</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_110">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>100</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Combustible:</string>
     </property>
    </widget>
    <widget class="QTableView" name="table_ComparativoEstaciones_Page5">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>30</y>
       <width>691</width>
       <height>451</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="BotonConsultar_Page5">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>200</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Consultar</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Ganancia por cada litro </string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_24">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>540</height>
     </rect>
    </property>
    <attribute name="label">
     <string>Cierre de turno: litros vendidos vs. diferencia de medidor y estado</string>
    </attribute>
    <widget class="QLabel" name="label_111">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Fecha de Turno:</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateTimeEdit_FechaTurno_Page6">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>160</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_112">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>220</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Numero de Turno:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="spinBox_NumTurno">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>230</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="suffix">
      <string/>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>99.000000000000000</double>
     </property>
    </widget>
    <widget class="QPushButton" name="BotonConsultar_Page6">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>290</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Consultar</string>
     </property>
    </widget>
    <widget class="QTableView" name="table_ComparativoEstaciones_Page6">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>30</y>
       <width>701</width>
       <height>441</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_129">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Estacion:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_Estacion_Page6">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_130">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Combustible:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_Combustible_Page6">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>90</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
